railties/lib/rails/application/bootstrap.rb

```diff
@@ -59,6 +59,14 @@ module Bootstrap
         unless config.consider_all_requests_local
           Rails.error.logger = Rails.logger
         end
+
+        if config.log_to_stdout? && !ActiveSupport::Logger.logger_outputs_to?(Rails.logger, STDOUT, STDERR)
+          console = ActiveSupport::Logger.new(STDOUT)
+          console.formatter = Rails.logger.formatter
+          console.level = Rails.logger.level
+
+          Rails.logger.extend(ActiveSupport::Logger.broadcast(console))
+        end
```

Comment:
This does seem to belong here with the other setup of the Rails.logger.
Also nice that it removes the duplication.